getting-started/dotnet/00.MIGRATION.md

```diff
@@ -1,20 +1,14 @@
-# Migrating from Botbuilder SDK to Teams AI.
+# Migrating from Bot Framework (BF)  SDK to Teams AI
```

Comment:
Would you mind adding a TOC for the `dotnet` and `js` directories, same format as `getting-started`? Thank you~

Reply:
done